### ISSUE TITLE
Set Parent Save button behavior while searching

### ISSUE THREAD

**Issue by zwarm** [wordpress-mobile/WordPress-Android#11318]
The goal of this issue is to do determine the proper behavior for showing the SAVE button while searching in the Set Parent feature. 

- If a parent is selected, then the user navigates to search, should the save button be visible?
![set-parent-search-save-behavior](https://user-images.githubusercontent.com/506707/74547574-54e10b00-4f1a-11ea-81f1-dbaf449dd163.gif)

**Comment by designsimply** [comment]
@osullivanchris can you answer this question for @zwarm?

**Comment by osullivanchris** [comment]
@designsimply @zwarm 
Hey, I just tested this and the save action does appear once you select a different page. I believe in the video above, the save action is not shown at that moment because "Moving tickets to the TOS queue" is already the parent.

Try testing it again but making the child page top-level again, and selecting a new parent. Then doing the same but changing it from one parent to another. I find it quite intuitive. The 'save' action is only appearing when I want to commit a change. It is not appearing when there is no change.

In the first screenshot below, the page has no existing parent. When I change parent from 'Top Level' to 'Contact', the save button appears to commit the change 
![without parent](https://user-images.githubusercontent.com/50576199/75057095-293cc280-54d0-11ea-869e-2887cd1b012b.png)

In the next screenshot, the page already has 'Contact' set as a parent, so no save button appears. When I tap on 'About', I get a save button
![with existing parent](https://user-images.githubusercontent.com/50576199/75057184-5e491500-54d0-11ea-88c1-c8b3e04c0c7f.png)

The same logic applies in search. First, I have made it top level again. When I search for contact, and choose it, a 'Save' action appears.
![Screenshot_20200221-173431](https://user-images.githubusercontent.com/50576199/75057293-96e8ee80-54d0-11ea-87ff-c9a724517da0.png)

However, with contact already selected as parent, if I search for it again, no action appears (below). When I search instead for 'About', the save action does appear, to change from one parent to another
![Screenshot_20200221-173639](https://user-images.githubusercontent.com/50576199/75057441-eb8c6980-54d0-11ea-99ea-2c561d7202eb.png)

Looking at all that I don't think this is a bug. We could add something to say 'this is already a parent' but I think its already working logically and felt reasonably intuitive to me
